frontend/components/droplets/lessons/table-renderer.tsx

```
@@ -0,0 +1,96 @@
+            <tr className="border-b border-slate-300 dark:border-slate-600">
+              {headerRow.cells.map((cell, cellIndex) => {
+                const bgColor = cell.backgroundColor || "inherit";
+                return (
+                  <th
+                    key={cellIndex}
+                    className="border border-slate-300 px-4 py-2 text-left font-semibold dark:border-slate-600"
+                    style={{
+                      backgroundColor:
+                        bgColor !== "inherit" ? bgColor : undefined,
+                    }}
+                    dangerouslySetInnerHTML={{ __html: cell.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)